pkg/kubectl/cmd/get/customcolumn.go

```diff
@@ -32,6 +32,7 @@ import (
 	"k8s.io/cli-runtime/pkg/printers"
 	"k8s.io/client-go/util/jsonpath"
 	utilprinters "k8s.io/kubernetes/pkg/kubectl/util/printers"
+	printerspkg "k8s.io/kubernetes/pkg/printers"
```

Comment:
You can't introduce this dependency. We're in the process of splitting kubectl out of main repo and this is conflicting with that direction. I'd suggest leaving `Column` definition where it is.